introduction.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsam eaque quidem quod nostrum sapiente, veniam consequatur, culpa corporis animi temporibus, sed inventore incidunt laudantium. Mollitia enim, fugit delectus repellendus esse, nisi corporis qui sunt minus voluptatibus alias porro libero iusto dolores, dignissimos suscipit sapiente architecto magni reiciendis nam. Dolorum doloribus fugiat consectetur aliquam harum, debitis nesciunt, dicta, exercitationem eius molestiae modi expedita quisquam illo amet soluta culpa porro autem ullam temporibus, ratione blanditiis quod vitae architecto eaque? Accusantium, non vitae provident tempore ipsa officiis reprehenderit! Soluta odio quae similique, quos magni est molestiae sed, ullam modi laborum quis dolorum sit!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.